composer.json

```diff
@@ -18,8 +18,8 @@
     }
   ],
   "require": {
-    "symfony/framework-bundle": "^2.0|^3.0",
-    "doctrine/orm": ">=2.2.3,<2.6-dev",
+    "symfony/framework-bundle": "^2.0 | ^3.0 | ^4.0",
+    "doctrine/orm": ">=2.2.3",
```

Comment:
Please do not use open constraints. `^2.2.3` should suffice, and exclude 3.0 (which will have a bunch of incompatible changes)